app/controllers/course/assessment/question/controller.rb

```diff
@@ -29,6 +32,18 @@ def authorize_create_question_in_koditsu
     raise CanCan::AccessDenied if @assessment.is_koditsu_enabled && is_course_koditsu_enabled
   end
 
+  def flag_question_not_synced_with_koditsu
+    return unless instance_of?(Course::Assessment::Question::ProgrammingController)
+
+    question = @programming_question.acting_as
+
+    question.update!(is_synced_with_koditsu: false)
+  end
+
+  def flag_assessment_not_synced_with_koditsu
+    @assessment.update!(is_synced_with_koditsu: false)
+  end
```

Comment:
use the `flag_not_synced_with_koditsu` method from `assessments_controller` instead of defining the same thing here.

Reply:
done